src/views/app/detail-panel/preview/attachments-block.tsx

```diff
@@ -433,8 +434,12 @@ const copyToFiles = (att: AttachmentPartType, message: MailMessage, nodes: any):
 
 const AttachmentsBlock: FC<{ message: MailMessage }> = ({ message }): ReactElement => {
 	const [expanded, setExpanded] = useState(false);
-	const attachments = useMemo(() => findAttachments(message?.parts, []), [message]);
-	const attachmentsCount = useMemo(() => attachments?.length, [attachments]);
+	const attachments = useMemo(
```

Comment:
as findAttachments function is not used any longer, it can be removed

Reply:
No it is used in other files also(managing inline) that is why I didn't removed it